data/json/items/armor/boots.json

```
@@ -1264,6 +1264,25 @@
     "material_thickness": 1,
     "flags": [ "VARSIZE", "SKINTIGHT" ]
   },
+  {
+    "id": "xlsocks",
+    "type": "ARMOR",
+    "name": { "str_sp": "pair of xl socks" },
+    "description": "Socks.  Big ones.  Put 'em on your feet.",
+    "weight": "58 g",
+    "volume": "250 ml",
+    "price": 200,
+    "price_postapoc": 50,
+    "material": [ "cotton" ],
+    "symbol": "[",
+    "looks_like": "socks_wool",
```

```suggestion
    "looks_like": "socks",
```
So the looks_like hierarchy looks for the original socks' texture before looking for the wool socks, if a tileset has a seperate image for regular ones.